core-services/prow/02_config/_config.yaml

```diff
@@ -1768,7 +1771,6 @@ tide:
     - openshift/ci-vm-operator
     - openshift/cincinnati-graph-data
     - openshift/cloud-ingress-operator
-    - openshift/cloud-provider-openstack
```

Comment:
In principle this PR is fine, I'd just expect *two* repos to be removed from this query and two added to each of the three queries above.

Reply:
yeah right there's something weird.

While I'm at it, do you have a clue what those `openshift-priv` repos are, and if they are supposed to exactly mirror their "priv-less" versions?

Reply:
https://docs.ci.openshift.org/docs/architecture/private-repositories/#openshift-priv-organization

The automation will reconcile all config for these to match the non-priv ones so there's no need to deal with them.